docs/source/examples/read-data-get-data.js

```diff
@@ -27,32 +27,34 @@ import {
   getUrlAll,
 } from "@inrupt/solid-client";
 
+import { FOAF } from "@inrupt/vocab-common-rdf";
```

Comment:
On the one hand I think it's good that we're showing people what @inrupt/vocab-common-rdf can do for them, but by relying solely on that we leave no breadcrumbs at all that can guide the user to what to do if the vocab they need is not part of that package. Something to ponder...

Reply:
Yup - so, I was trying that by leaving the comments below with the "representing the http://xmlns.com/foaf/0.1/<x>" but that seems to have missed the mark.  Will definitely ponder over a better way.